frontend/src/metabase/collections/components/BaseTableItem.jsx

```diff
@@ -59,47 +57,102 @@ export function BaseTableItem({
   const handleCopy = useCallback(() => onCopy([item]), [item, onCopy]);
   const handleArchive = useCallback(() => item.setArchived(true), [item]);
 
-  const testId = isPinned ? "pinned-collection-entry" : "collection-entry";
+  const renderRow = useCallback(() => {
+    const canSelect = typeof onToggleSelected === "function";
+
+    const lastEditInfo = item["last-edit-info"];
+
+    // We don't keep last edit info for pulses
+    // TODO Remove ternary when Pulses are gone
```

Comment:
To prevent TODOs from living forever in the codebase, a suggestion:
create or link a Github issue about this.

We can keep the TODO if needed.
Just slightly more likely that it will be solved in the future if it looks like

```js
// TODO Remove ternary when Pulses are gone (metabase#16948)
```